src/App.js

```diff
@@ -1,13 +1,64 @@
 import React from 'react';
-
 import './App.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Button } from 'react-bootstrap';
+import { getAll, get5First, getRed } from './api/goods';
+import GoodList from './components/GoodsList';
+
+class App extends React.Component {
+  state = {
+    products: null,
+  }
+
+  allHandler = async() => {
+    this.setState({
+      products: await getAll(),
+    });
+  }
+
+  fiveHandler = async() => {
+    this.setState({
+      products: await get5First(),
+    });
+  }
+
+  redHandler = async() => {
+    this.setState({
+      products: await getRed(),
+    });
+  }
```

Comment:
could you make 1 function instead of these 3, that accepts a callback as a parameter?